### ISSUE TITLE
The pretrained fastscnn model

### ISSUE THREAD

**Issue by CuttlefishXuan** [open-mmlab/mmsegmentation#350]
Hi, what's the batch size and learning rate for the pretrained fast-scnn model from here: https://download.openmmlab.com/mmsegmentation/v0.5/fast_scnn/fast_scnn_4x8_80k_lr0.12_cityscapes-f5096c79.pth

And for the model file's name, what's the meaning of "4x8"?

**Comment by Junjun2016** [comment]
4 GPUs and 8 images per GPU.

**Comment by CuttlefishXuan** [comment]
ok, thanks
